src/scss/components/_dialog.scss

```diff
@@ -47,12 +55,35 @@
     &-name {
       line-height: 15px;
       font-weight: bold;
-      padding-top: 4px;
     }
 
     &-location {
       font-size: 12px;
       line-height: 15px;
     }
   }
+
+  &--inverted {
```

Comment:
@iondrimba `--inside-wall` could be more explicit here instead `inverted`. Wall is a container component that have a background-color rastaGray. What do you think?